src/librustc_trans/save/dump_csv.rs

```diff
@@ -70,6 +72,9 @@ pub struct DumpCsvVisitor<'l, 'tcx: 'l> {
     fmt: FmtStrs<'l, 'tcx>,
 
     cur_scope: NodeId,
+
+    // Map of macro definition spans to the set of (known) callsite spans.
```

Comment:
That should be impossible (at least for valid spans). Of course, knowing the macro stuff I can't make any guarantees. I'll rewrite it to have two sets anyway (one can make the argument we can probably ignore malformed cases anyway)